hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/SequenceFile.java

```diff
@@ -1051,6 +1051,11 @@ public static Option compression(CompressionType value,
       return new CompressionOption(value, codec);
     }
 
+    @org.apache.hadoop.thirdparty.com.google.common.annotations.VisibleForTesting
```

Comment:
I'm proposing elsewhere to not use mockito, which avoids exposing this operation...I'd be worried about exposing something into a broadly used API, as inevitably someone will use it in production

Reply:
Makes sense.